opentracing/tracer.py

```diff
@@ -82,3 +83,24 @@ def close(self):
         fut = Future()
         fut.set_result(True)
         return fut
+
+    def implementation_id(self):
+        """Returns the ImplementationID for this OpenTracing implementation.
+
+        :return: An ImplementationID instance that describes this OpenTracing
+            implementation.
+        """
+        return ImplementationID('noop', '1.0.0')
+
+
+# Per collections.namedtuple, the below defines a class called ImplementationID
+# which can be initialized via positional parameters or a kwargs-style
+# initializer.
+#
+#     dapper_impl = ImplementationID('dapper', '0.1.2')
+#     zipkin_impl = ImplementationID(name='zipkin', version='0.3.4')
+#
+ImplementationID = namedtuple('ImplementationID', 'name version')
+
+# The SemVer (http://semver.org/) of the OpenTracing Python API.
+OPENTRACING_PYTHON_SEMVER = '0.9.0'
```

Comment:
I would put it at the top of the file.

> the biggest challenge will be remembering to update it

:-)  I remember seeing somewhere a lib was taking a hash of its public API and having a unit test comparing that hash with a constant, and a comment in upper case "if this hash changed the version needs to change". If I have any grey hair, that's when it started.